solutions/core-landing-zone/setters.yaml

```diff
@@ -122,6 +124,16 @@ data:
   # customization: The values below must be modified to locked: true and retentionDays: 365 in a Production setting to implement above mentioned security controls.
   retention-locking-policy: "false"
   retention-in-days: "1"
```

Comment:
to follow the latest standards, please add a line that only contains a "#" between sections

Reply:
I changed a few things around; let me know what you think. Please resolve if satisfied.